request_handlers.cc

```diff
@@ -184,20 +185,21 @@ RequestHandler::Status StatusHandler::handleRequest(const Request& request,
   }
   reply += "\n";
 
-  std::string table = "Count | URL Requested | Status Code\n"
-                      "-----------------------------------\n";
+  std::string table = "Count | URL Requested                                   | Status Code\n"
+                      "---------------------------------------------------------------------\n";
   const char separator = ' ';
-  const int count_width = 6;
-  const int url_width = 14;
+  const int count_width = 8;
+  const int url_width = 50;
   const int status_width = 11;
-  std::stringstream line;
 
+  reply += table;
   for (auto it : tuple_to_count) {
-    line << std::to_string(it.second) << std::setw(count_width) << "| "
-         << it.first[0] << std::setw(url_width) << "| "
-         << it.first[1] << std::setw(status_width) << "\n";
+    std::stringstream line;
```

Comment:
For documentation: the `std::stringstream` was too long-lived, and even our call to `clear()` didn't actually clear it between uses.